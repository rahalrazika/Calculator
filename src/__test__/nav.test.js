import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import Nav from '../components/nav';

it('expect to render Nav components', () => {
  const result = renderer.create(
    <Router>
      <Nav />
    </Router>,
  ).toJSON;
  expect(result).toMatchSnapshot();
});
