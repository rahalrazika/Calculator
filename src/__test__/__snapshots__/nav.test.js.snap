// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Nav components 1`] = `[Function]`;

exports[`expect to render home components 1`] = `[Function]`;
